test/array.test.js

```diff
@@ -210,6 +210,69 @@ buildTest({
   foo: [1, 'string', {}, null]
 })
 
+test('array items is a list of schema and additionalItems is true, just the described item is validated', (t) => {
+  t.plan(1)
+
+  const schema = {
+    type: 'object',
+    properties: {
+      foo: {
+        type: 'array',
+        items: [
+          {
+            type: 'string'
+          }
+        ],
+        additionalItems: true
+      }
+    }
+  }
+
+  const stringify = build(schema)
+  const result = stringify({
+    foo: [
+      'foo',
+      'bar',
+      1
+    ]
+  })
+
+  t.equal(result, '{"foo":["foo","bar",1]}')
+})
+
+test('array items is a list of schema and additionalItems is false', (t) => {
+  t.plan(1)
+
+  const schema = {
+    type: 'object',
+    properties: {
+      foo: {
+        type: 'array',
+        items: [
+          {
+            type: 'string'
+          }
+        ],
+        additionalItems: false
+      }
+    }
+  }
+
+  const stringify = build(schema)
+
+  try {
+    stringify({
+      foo: [
+        'foo',
+        'bar'
+      ]
+    })
+    t.fail()
+  } catch (error) {
+    t.ok(/does not match schema definition./.test(error.message))
```

Comment:
This matches the definition. Should fail if it was `{foo:['foo','bar',1]}`

Reply:
Reference: https://json-schema.org/understanding-json-schema/reference/array.html
The `additionalItems` keyword controls whether it’s valid to have additional items in the array beyond what is defined in `items`.

Reply:
@zekth The case pass because additionItems is 'false', and so the schema expect that the array foo should has just one string item.

Reply:
Missed this line. Thanks for clarify.
> Tuple validation is useful when the array is a collection of items where each has a different schema and the ordinal index of each item is meaningful.

Reply:
@zekth Do you still need some changes?

Reply:
Thanks for the clarification.

Reply:
It's a must. I want to contribute, but I don't want you lose much time to understand what I did.

I'll try to be more clear in PR description next time :)